modules/apps_api/lib/apps_api/notification_service.rb

```diff
@@ -89,16 +99,26 @@ def event_unsuccessful?(event)
          event['target'][0]['detailEntry']['subject'].nil?)
```

Comment:
worth putting some nil checking in here?

Reply:
can you rephrase this? I'm not quite sure what you're asking. The target will always have at least one item because an event is only created (of the event types we are looking for) when there is both a `subject` and a `target` in the event data. There may be more than one `target`, but never less. This nil check is to handle the below edge case from talking with okta.

> Not all disconnect events will have a target[0]['detailEntry']['subject'] present in the hash. This seems to happen when the user being disconnected has already been deleted from Okta. It's an edge case but it is safe to assume any events without this key can be ignored.

Reply:
ok, sounds good, i thought there may have been a strange edge case where that list of targets could be empty.
sounds like there isn't, fair enough! 👍🏻